config/default.js

```diff
@@ -105,4 +105,5 @@ module.exports = {
     APP_ACCESS_KEY: "EVENT_100MS_APP_ACCESS_KEY",
     APP_SECRET: "EVENT_100MS_APP_SECREt",
```

Comment:
I think there's a small typo in the Event100ms section; `APP_SECREt` should be corrected to `APP_SECRET.`

Reply:
> I think there's a small typo in the Event100ms section; `APP_SECREt` should be corrected to `APP_SECRET.`
😵

Reply:
> I think there's a small typo in the Event100ms section; `APP_SECREt` should be corrected to `APP_SECRET.`

Are you sure it is a typo and I made these changes?

Reply:
I made these changes?